src/classes/C501_CTRL_ClassReg_Home.cls

```diff
@@ -0,0 +1,48 @@
+public class C501_CTRL_ClassReg_Home {
+    public String	acctID {get; set;}
+    
+    public C501_CTRL_ClassReg_Home(){
+        acctID	= null;
+        
+        /*Cookie aId = ApexPages.currentPage().getCookies().get('AccountIdClassRegSite');
+        if (aId != null){
+            acctID = 'cookie found';//aId.getValue(); 
+        } else {
+            String acctPa = ApexPages.currentPage().getParameters().get('a');
+            String secret = ApexPages.currentPage().getParameters().get('s');
+            String rember = ApexPages.currentPage().getParameters().get('r');
+            if(acctPa != null && secret != null){
+                Datetime 	thirtyMinutesAgo	= Datetime.now().addMinutes(-30);
+                List<Class_Site_Login_Attempt__c> attList = [
+                    SELECT 	id,
```

Comment:
Can you explain why you chose this format for SOQL vs default supported in visual studio code?  Adding the extra tabs/spaces takes extra time to keep everything lined up.

Here is what I generally do for SOQL - and if need a new line for more parameters that is indented by 4 spaces (tab) under the previous line.

                    SELECT Merge_Metric__r.Name
                    FROM C501_Account_Merge__c
                    GROUP BY Merge_Metric__r.Name
                    ORDER BY Merge_Metric__r.Name ASC];

Reply:
This is my own personal format for SOQL, which helps me see at a glance everything about the query. Notice how the indents show the items being acted on are all in the same column and the control words are in the same column. 

Again, didn't create this on Code.

Reply:
Lets agree on a format and then we'll both can use the same format and we can work towards a 501 commons coding standards document.  This will be the first thing in the document that we talk about is SOQL code formatting.

Here is two different examples from V4SF which is the same format I follow.  I tend to put each SOQL statement on a new line that is indented by 4 spaces

within for - where on next line indented by 4 spaces

                for (Volunteer_Job__c vj : [select Name, Id from Volunteer_Job__c 
                    where Campaign__r.IsActive = true order by name limit 999]) {
                    listSO.add(new SelectOption(vj.id, vj.name));
                }           

variable assignment - if they do a new line is indented by 4 spaces

        list<Volunteer_Shift__c> listShift = [select Id, Name, Volunteer_Job__r.Campaign__c, Start_Date_Time__c 
        	from Volunteer_Shift__c where Volunteer_Job__r.Campaign__c = :cmp2.Id];